Dockerfile

```diff
@@ -86,6 +88,14 @@ RUN git clone https://github.com/codemasher/php-ext-xz.git --branch ${PHP_EXT_XZ
   && phpize \
   && ./configure \
   && make && make install
+  
+## Zopfli Extension
+FROM compile AS zopfli
+RUN git clone https://github.com/artfulhacker/php_zopfli.git \
```

Comment:
This extension is really old. Let's use [this](https://github.com/kjdev/php-ext-zopfli) one instead.

Reply:
I have made the changes with the suggested one. @stnguyen90